data/crac-file/crac-io-json/src/test/resources/file.json

```diff
@@ -0,0 +1,424 @@
+          "direction" : "OUT",
+          "maxValue" : 999.0
+        },
+        "state" : {
+          "@c" : "com.farao_community.farao.data.crac_impl.SimpleState",
+          "contingency" : {
+            "@c" : "com.farao_community.farao.data.crac_impl.ComplexContingency",
+            "id" : "idContingency",
+            "name" : "My contingency",
+            "networkElements" : [ {
+              "@c" : ".NetworkElement",
+              "id" : "idLine2",
+              "name" : "My Line 2"
+            }, {
+              "@c" : ".NetworkElement",
+              "id" : "idLine3",
+              "name" : "My Line 3"
+            }, {
+              "@c" : ".NetworkElement",
+              "id" : "idNE1",
+              "name" : "My Element 1"
+            } ]
+          },
+          "instant" : {
+            "@c" : ".Instant",
+            "duration" : 200.0
+          }
+        }
+      }
+    }, {
+      "@c" : "com.farao_community.farao.data.crac_impl.usage_rule.OnContingency",
+      "usageMethod" : "FORCED",
+      "state" : {
+        "@c" : "com.farao_community.farao.data.crac_impl.SimpleState",
+        "contingency" : {
+          "@c" : "com.farao_community.farao.data.crac_impl.ComplexContingency",
+          "id" : "idContingency",
+          "name" : "My contingency",
+          "networkElements" : [ {
+            "@c" : ".NetworkElement",
+            "id" : "idLine2",
+            "name" : "My Line 2"
+          }, {
+            "@c" : ".NetworkElement",
+            "id" : "idLine3",
+            "name" : "My Line 3"
+          }, {
+            "@c" : ".NetworkElement",
+            "id" : "idNE1",
+            "name" : "My Element 1"
+          } ]
+        },
+        "instant" : {
+          "@c" : ".Instant",
+          "duration" : 200.0
+        }
+      },
+      "contingency" : {
+        "@c" : "com.farao_community.farao.data.crac_impl.ComplexContingency",
+        "id" : "idContingency",
+        "name" : "My contingency",
+        "networkElements" : [ {
+          "@c" : ".NetworkElement",
+          "id" : "idLine2",
+          "name" : "My Line 2"
+        }, {
+          "@c" : ".NetworkElement",
+          "id" : "idLine3",
+          "name" : "My Line 3"
+        }, {
+          "@c" : ".NetworkElement",
+          "id" : "idNE1",
+          "name" : "My Element 1"
+        } ]
+      }
+    } ],
+    "ranges" : [ {
+      "@c" : "com.farao_community.farao.data.crac_impl.range_domain.AbsoluteFixedRange",
+      "min" : 10.0,
+      "max" : 1000.0
+    }, {
+      "@c" : "com.farao_community.farao.data.crac_impl.range_domain.RelativeDynamicRange",
+      "min" : 100.0,
+      "max" : 1000.0
+    }, {
+      "@c" : "com.farao_community.farao.data.crac_impl.range_domain.RelativeFixedRange",
+      "min" : 1.0,
+      "max" : 10.0
+    } ],
+    "applicableRangeActions" : [ {
+      "@c" : "com.farao_community.farao.data.crac_impl.remedial_action.range_action.PstRange",
+      "networkElement" : {
```

Comment:
same remark for network Elements within RA

Reply:
same